muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/actions/base/status.ts

```diff
@@ -64,6 +65,8 @@ async function loadWhoAMI(
 
   const whoAmI = await userApi.getWhoAmI();
 
+  const isStudent = whoAmI.roles ? whoAmI.roles.includes(Role.Student) : false;
```

Comment:
Unnecessary null/undefined check as roles is always empty list by default. Can use includes checking straight